#!/usr/bin/env python2
from dynamic_reconfigure.parameter_generator_catkin import *
PACKAGE = "asr_ism"


gen = ParameterGenerator()

gen.add("capture_interval", int_t, 0, "Duration of asynchronous recognizer result buffering.", 3, 0, 120)
gen.add("markerlifetime", double_t, 0, "Time Markers are shown (0=permanent).", 0, 0, 120)
gen.add("posePredictionSampelFaktor", double_t, 0, "Samplerate of Poseprediction.", 10, 0, 100)
gen.add("validPositionVis", bool_t, 0, "Visualize room of possible positions.", False)

exit(gen.generate(PACKAGE, "asr_ism", "recognizer"))
